maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/EnforcerMojo.java

```diff
@@ -168,6 +177,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             case "typos":
                 runTypos(filesToCheck, failuresByRule, warningsByRule);
                 break;
+            case "inclusiveNaming":
```

Comment:
I would prefer `inclusive-naming` in maven pom file rather than camel case (@romain-grecourt what do you think?)

Reply:
I do prefer `-` separated over camel case.

Reply:
Done